app/src/main/java/uco_396035/movio2/pv256/fi/muni/cz/movio/MainActivity.java

```diff
@@ -8,6 +8,15 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        changeTheme();
```

Comment:
toto nemoze fungovat, aspon mne to teda nefunguje.
pretoze onCreate sa vola iba pri vzniku aktivity, kedy sa uz aktivita inicializuje s preddefinovanou temou, takze getTheme bude mat vzdy rovnaku hodnotu...
je potrebne poslednu nastavenu temu ukladat aspon do preferences a potom na stisk tlacitka alebo na spustenie aktivity checkovat co bolo naposledy nastavene v preferences.

Reply:
a tiez to musi byt definovane este pred onCreate, aby sa na layout definovany v xml uz aplikovala ta tema ktoru chceme nastavit.